tests/retryReads-move-sharded.yaml

```diff
@@ -0,0 +1,62 @@
+initialConfiguration:
+  clusterConfiguration:
+    clusterType: SHARDED
+    providerSettings:
+      providerName: AWS
+      regionName: US_EAST_1
+      instanceSizeName: M10
+  processArgs: {}
+
+operations:
+  -
+    setClusterConfiguration:
+      clusterConfiguration:
+        providerSettings:
+          providerName: AWS
+          regionName: US_EAST_1
+          instanceSizeName: M20
+      processArgs: {}
+
+uriOptions:
+  retryReads: true
```

Comment:
This is something I would have expected in the `createEntities` section of the `driverWorkload` object.

Even if Astrolabe were to process `initialData` I don't think these options would be relevant.

Reply:
Moved all of the uri options into createEntities and removed their handling from astrolabe entirely.